tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py{27,35,36}-django{111},
```

Comment:
Isn't it a good time to remove python2 and 3.5 ?

Reply:
https://github.com/peopledoc/django-agnocomplete/issues?q=is%3Aissue+is%3Aopen+drop ;o)